app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.android.example.model
+
+import com.google.gson.annotations.SerializedName
+import org.joda.time.format.DateTimeFormat
+import org.ocpsoft.prettytime.PrettyTime
+import java.util.Locale
+
+// class News(var author: String, var newsText: String, var time: String)
+
+data class News(
+    var id: Int,
+    @SerializedName("userId")
+    var userId: Int,
+    @SerializedName("createdAt")
+    var createdAt: String,
+    var title: String,
+    @SerializedName("picture")
+    var picture: String,
```

Comment:
The serialize name is the same as the field. I think that it should work without the annotation